src/methods/tx/_events.ts

```diff
@@ -231,9 +249,25 @@ export const processChainEvent = (
         break;
       case 'IntentionResolvedDirectTradeFees':
         /**
-         * parsedData: <Array> [AccountId, AccountId, AssetId, Balance]
-         *                     [who, account paid to, asset, fee amount]
+         * parsedData: <Array> [AccountId, IntentionID, AccountId, AssetId, Balance]
+         *                     [who, IntentionID, account paid to, asset, fee amount]
          */
+        if (Array.isArray(parsedData)) {
+          mergeEventToScope({
+            ...exchangeTxEventData,
+            data: {
+              id: parsedData[1]?.toString(),
+              fees: [
+                {
+                  account1: parsedData[0]?.toString(),
+                  account2: parsedData[2]?.toString(),
+                  asset: parsedData[3]?.toString(),
+                  amount: new BigNumber(parsedData[4]?.toString()),
+                },
```

Comment:
It seems parsedData is used in wrong way.
According to the comment above, accountId seems the first element of the array, but parsedData[1] is assigned to id and parsedData[0] is assigned to account1.
Is it intentional or am I misunderstanding something?

Reply:
Hm... not sure that I see any issue here:
```
// parsedData: <Array> [AccountId, IntentionID, AccountId, AssetId, Balance]
                                        [account1,   id,                account2,  asset,     amount]

mergeEventToScope({
            ...exchangeTxEventData,
            data: {
              id: parsedData[1]?.toString(),
              fees: [
                {
                  account1: parsedData[0]?.toString(),
                  account2: parsedData[2]?.toString(),
                  asset: parsedData[3]?.toString(),
                  amount: new BigNumber(parsedData[4]?.toString()),
                },
              ],
            },
          });
```